lib/srv/usermgmt_impl.go

```diff
@@ -0,0 +1,28 @@
+//go:build !linux
```

Comment:
Other files in the codebase with negative build constraints are typically called `_other` (with one `_stub`); let's not add yet another naming scheme 😅